tests/shape/rectangular_tag.lua

```diff
@@ -0,0 +1,13 @@
+local shape,cr,show = ... --DOC_HIDE
+ 
+shape.rectangular_tag(cr, 70, 70)
+show(cr) --DOC_HIDE
+ 
+-- shape.rectangular_tag(cr,20,70) --TODO broken
```

Comment:
This one shows up in the generated API docs (and sounds like something that should be fixed).